tests/ops/service/connectors/test_bigquery_queryconfig.py

```diff
@@ -108,6 +116,35 @@ def test_generate_query_with_namespace_meta(
             == expected_query
         )
 
+    def test_generate_query_with_multiple_identities(
+        self, execution_node: ExecutionNode
+    ):
+        query_config = BigQueryQueryConfig(execution_node)
+        assert (
+            query_config.generate_query(
+                input_data={
+                    "email": ["customer-1@example.com", "customer-2@example.com"]
+                }
+            ).text
+            == "SELECT address_id, created, custom_id, email, extra_address_data, id, name FROM `customer` WHERE (email IN (:email_in_stmt_generated_0, :email_in_stmt_generated_1))"
+        )
+
+    def test_generate_query_with_nested_identity(
+        self, customer_profile_node: ExecutionNode
+    ):
+        query_config = BigQueryQueryConfig(customer_profile_node)
+        assert (
+            query_config.generate_query(
+                input_data={
+                    "contact_info.primary_email": [
+                        "customer-1@example.com",
+                        "customer-2@example.com",
+                    ],
+                }
+            ).text
+            == "SELECT address, contact_info, id FROM `customer_profile` WHERE (contact_info.primary_email IN (:contact_info_primary_email_in_stmt_generated_0, :contact_info_primary_email_in_stmt_generated_1))"
```

Comment:
Unit test for `generate_query`, you can see the `IN` clause doesn't have periods in the placeholders `:contact_info_primary_email_in_stmt_generated_0` and `:contact_info_primary_email_in_stmt_generated_1`